[2023-01-08T15:06:21.269+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.upload_common_modules_py_to_s3 manual__2023-01-08T15:06:09.206673+00:00 [queued]>
[2023-01-08T15:06:21.280+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.upload_common_modules_py_to_s3 manual__2023-01-08T15:06:09.206673+00:00 [queued]>
[2023-01-08T15:06:21.280+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-01-08T15:06:21.280+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 3
[2023-01-08T15:06:21.280+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-01-08T15:06:21.296+0000] {taskinstance.py:1389} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_common_modules_py_to_s3> on 2023-01-08 15:06:09.206673+00:00
[2023-01-08T15:06:21.302+0000] {standard_task_runner.py:52} INFO - Started process 255 to run task
[2023-01-08T15:06:21.305+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'atn_update_datalake', 'upload_common_modules_py_to_s3', 'manual__2023-01-08T15:06:09.206673+00:00', '--job-id', '4034', '--raw', '--subdir', 'DAGS_FOLDER/market_data_analytics/my_dags/datalake_update.py', '--cfg-path', '/tmp/tmp_vq2jtk5', '--error-file', '/tmp/tmp0deyr8bj']
[2023-01-08T15:06:21.306+0000] {standard_task_runner.py:80} INFO - Job 4034: Subtask upload_common_modules_py_to_s3
[2023-01-08T15:06:21.372+0000] {task_command.py:371} INFO - Running <TaskInstance: atn_update_datalake.upload_common_modules_py_to_s3 manual__2023-01-08T15:06:09.206673+00:00 [running]> on host 370ed6868bc3
[2023-01-08T15:06:21.521+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=atn
AIRFLOW_CTX_DAG_ID=atn_update_datalake
AIRFLOW_CTX_TASK_ID=upload_common_modules_py_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-01-08T15:06:09.206673+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-08T15:06:09.206673+00:00
[2023-01-08T15:06:21.578+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-08T15:06:21.579+0000] {connection_wrapper.py:210} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2023-01-08T15:06:22.811+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=atn_update_datalake, task_id=upload_common_modules_py_to_s3, execution_date=20230108T150609, start_date=20230108T150621, end_date=20230108T150622
[2023-01-08T15:06:22.862+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-08T15:06:22.938+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
